dsl/Env/GitSsh.dhall

```diff
@@ -0,0 +1,5 @@
+-- Git configuration schema to access the git repo 
+{-
+    SSH configuration to access git repository 
```

Comment:
d: can you add comments on the properties? i.e. what is "local", what is remote? what format does the ssh-key need to be in? this will help users